CHANGELOG.md

```diff
@@ -738,4 +747,5 @@ thrown to caller to be dealt with so no need to log at this verbosity by pgjdbc
 [42.3.6]: https://github.com/pgjdbc/pgjdbc/compare/REL42.3.6...REL42.4.0
 [42.4.0]: https://github.com/pgjdbc/pgjdbc/compare/REL42.4.0...REL42.4.1
 [42.4.1]: https://github.com/pgjdbc/pgjdbc/compare/REL42.4.1...REL42.4.2
-[Unreleased]: https://github.com/pgjdbc/pgjdbc/compare/REL42.4.2...HEAD
+[42.4.2]: https://github.com/pgjdbc/pgjdbc/compare/REL42.4.2...REL42.5.0
+[Unreleased]: https://github.com/pgjdbc/pgjdbc/compare/REL42.5.0...HEAD
```

Comment:
I have missed this in previous versions but the comparison order should be:

`[VERSION]: .../compare/REL_PREVIOUS_VERSION...REL_VERSION` (the final REL should match the linked version)

We have a duplicate version 42.3.4, and from there the versions are mixed.

Deleting all the lines since 42.3.4 (including the duplicate) and adding this should fix it:

```
[42.3.4]: https://github.com/pgjdbc/pgjdbc/compare/REL42.3.3...REL42.3.4
[42.3.5]: https://github.com/pgjdbc/pgjdbc/compare/REL42.3.4...REL42.3.5
[42.3.6]: https://github.com/pgjdbc/pgjdbc/compare/REL42.3.5...REL42.3.6
[42.4.0]: https://github.com/pgjdbc/pgjdbc/compare/REL42.3.6...REL42.4.0
[42.4.1]: https://github.com/pgjdbc/pgjdbc/compare/REL42.4.0...REL42.4.1
[42.4.2]: https://github.com/pgjdbc/pgjdbc/compare/REL42.4.1...REL42.4.2
[42.5.0]: https://github.com/pgjdbc/pgjdbc/compare/REL42.4.2...REL42.5.0
[Unreleased]: https://github.com/pgjdbc/pgjdbc/compare/REL42.5.0...HEAD
```